src/vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewServiceImpl.ts

```diff
@@ -302,7 +311,14 @@ window.onload = function() {
 	});
 };
 </script>`);
-		return new NotebookOutputWebview(id, runtime.runtimeMetadata.runtimeId, webview);
+
+		return new NotebookOutputWebview(
+			id,
+			runtime?.runtimeMetadata.runtimeId ?? '',
```

Comment:
Is this safe? At a quick glance, `INotebookOutputWebview.sessionId` is used in `WebviewPlotClient.metadata` which is used to store some state:

https://github.com/posit-dev/positron/blob/3ed030d9c0cf5a7fb59824d290b8a596e99b3c60/src/vs/workbench/contrib/positronPlots/browser/positronPlotsService.ts#L435

Reply:
Good catch. I totally forgot to go in and fix that. Updated so if you don't supply a runtime you have to supply some sort of session id string.